source/default/_patterns/00-protons/legacy/css/pages/utc_homepages_options.css

```diff
@@ -89,4 +89,24 @@ p.small-head-hr {
     .utc-homepage-quote-bg blockquote {
         width: 100%;
     }
+}
+
+
+/**augment tailwinds bg-opacity-20**/
+
+.bg-opacity-20 {
+    opacity: .2!important
+}
+
+.bg-opacity-15 {
+    opacity: .15!important
+}
+
+@media screen and (min-width:1024px) {
```

Comment:
https://tailwindcss.com/docs/breakpoints,
https://tailwindcss.com/docs/container
I'm pretty sure this changes can be made with tailwind itself.
Im a bit concern about doing attaching to section and class names, because if we refactor and took one of those out later it be hard to tell why a certain class does not apply anymore.
It seems like background opacity can be done as well. @brittrich 
https://tailwindcss.com/docs/background-opacity
Good work though.

Reply:
Do you have a suggestion as to how to target just the heros added to a "boxed" section in laybuilder? The classes that differentiate between full and boxed are ".container-full" and ".container" respectively. That's why I attached the css to the the .container.

Reply:
So, when I was looking at this, I had the same issue. I couldn't figure out how to target boxes inside the container without using css classes. So if we don't want to add additional css....could we justadd margin regardless of container and width? I don't know that it really harms the look of the full width banner?

<img width="1741" alt="Screen Shot 2021-07-01 at 9 49 35 PM" src="https://user-images.githubusercontent.com/50490141/124208804-5af76d80-dab6-11eb-83db-9c3ce1330ba2.png">

Reply:
You beat me to the punch! I'm game! I think this is a great solution.

Reply:
It's not necessarily an issue with adding the css or not. I'm sure in some cases that's the best case scenario. The problem is that if we do it like that, we will be dependent on those other classes to exists. The minute we refactor a class or switch a class name (upstream container or something like it) the css class won't apply anymore.
Drupal lets us change all css classes in twig but that also means specificity on css classes sometimes could stop working because of it. We had a bug last year related to something like it. 

Finding that your css style is not loading anymore because one class is missing is one tricky bug to debug.

That said, that's an interesting solution. I like how thinking trough it as a group helps us narrow down some concerns and find innovate or overlooked approaches that achieve the same outcome but made the code DRY, easier to maintain and more modular.

Reply:
@bmartinez287, makes sense! I've got a PR on the way that will give us something with margin regardless of container size and keep us within Tailwind. I'll work on adding the suggested color option as well @UTCbridget. I took a closer look at how the grid was working and I think there was some improving to do there anyways...now that I'm understand "grid" + TW more thoroughly.